src/pepr/operator/controllers/keycloak/authservice/authservice.spec.ts

```diff
@@ -3,15 +3,316 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
  */
 
-import { beforeEach, describe, expect, test, vi } from "vitest";
-import { UDSPackage } from "../../../crd";
-import { updateCfg, updateCfgSecrets } from "../../config/config";
+import { beforeEach, describe, expect, it, test, vi } from "vitest";
+import { Sso, UDSPackage } from "../../../crd";
+import { AuthserviceClient, Mode } from "../../../crd/generated/package-v1alpha1";
+import { cleanupWaypointLabels } from "../../istio/ambient-waypoint";
+import { getWaypointName } from "../../istio/waypoint-utils";
 import { Client } from "../types";
-import { updatePolicy } from "./authorization-policy";
-import { buildChain, buildConfig } from "./authservice";
-import { initializeOperatorConfig } from "./config";
+import * as authorizationPolicy from "./authorization-policy.js";
+import { authservice, buildChain, buildConfig } from "./authservice.js";
```

Comment:
nit on import style to match here:
```suggestion
import * as authorizationPolicy from "./authorization-policy";
import { authservice, buildChain, buildConfig } from "./authservice";
```